### PR TITLE
[doc] Add documentation for tracking multiple values at once

### PR DESCRIPTION
**Pull request by alberttorosyan**

### CODE REVIEW

**Comment by alberttorosyan** [comment]
docs for #2087

File: docs/source/quick_start/supported_types.md [hunk 1]
```diff
@@ -20,6 +20,32 @@ provided by the package:
 - [Text](#text-tracking-with-aim)
 - [Figure](#figure-tracking-with-aim)
 
+### Tracking multiple values
+Starting from `v3.14.0` Aim provides ability to track multiple values at once. If the context and epoch are the same,
```
Comment (gorarakelyan) [inline docs/source/quick_start/supported_types.md:24]: Should the version be mentioned or are the docs already versioned by rtd?

File: docs/source/quick_start/supported_types.md [hunk 2]
```diff
@@ -20,6 +20,32 @@ provided by the package:
...
+the dictionary can be passed and the keys will be treated as names for tracked values. For example the code snippet below
```
Comment (gorarakelyan) [inline docs/source/quick_start/supported_types.md:25]: suggestion:

To track multiple values of different sequences but of the same context, just pass a dictionary of `<name: value>` pairs to the `track` method:

```py
aim_run.track({'accuracy': 98.2, 'loss': 0.001}, context={'subset': 'train'}, step=10, epoch=1)
```

Note: `name` argument should be set to `None` in this case.

further examples ...

File: docs/source/quick_start/supported_types.md (hunk 1 above)
Comment (alberttorosyan) [reply docs/source/quick_start/supported_types.md:24]: @gorarakelyan I think we should. It's a fast way to know when the feature first appears, regardless the current latest version.

Comment (gorarakelyan) [reply docs/source/quick_start/supported_types.md]: Is it a common practice? 
Having properly versioned docs should be enough imo.

Comment (mihran113) [reply docs/source/quick_start/supported_types.md]: @gorarakelyan for example official python docs do that:
https://docs.python.org/3/library/datetime.html#datetime.datetime.fromisoformat

**Review by mihran113 — APPROVED** [review]
lgtm

File: docs/source/quick_start/supported_types.md (hunk 1 above)
Comment (alberttorosyan) [reply docs/source/quick_start/supported_types.md]: @gorarakelyan @mihran113 if we decide to mention the version in the docs, we can choose the format, as for now such lines are in multiple places and there's no structured approach to this

**Review by gorarakelyan — APPROVED** [review]

**Pull request merged**